test/plugin/scenarios/kafka-scenario/support-version.list

```diff
@@ -25,3 +25,6 @@
 2.6.1
 2.7.0
 2.8.0
+3.0.2
+3.1.2
+3.2.3
```

Comment:
It looks like you are adding supports to 3.x, not just 3.2.x. Please update the PR title.

Reply:
Fixed